### PR TITLE
Localization markup extension

### PR DESCRIPTION
**Pull request by szv**
Did you think about using a _Custom Markup Extension_ for receiving the localized strings?
This would thin-out the view-models a little bit.

What do you think about that?

### CODE REVIEW

**Comment by morning4coffe-dev** [comment]
I love it! Simpler than my previous solution and works on every platform as well. Thanks, I have moved the other strings also.

File: ProjectSBS/ProjectSBS/Presentation/CustomMarkupExtensions/Localize.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+[MarkupExtensionReturnType(ReturnType = typeof(string))]
+internal class Localize : MarkupExtension
+{
+    public string? Name { get; set; }
+
+    protected override object ProvideValue() =>
+        string.IsNullOrEmpty(Name)
+        ? string.Empty
+        : Name.Localize();
```
Comment (morning4coffe-dev) [inline ProjectSBS/ProjectSBS/Presentation/CustomMarkupExtensions/Localize.cs:14]: A small improvement I was thinking about here would be to throw an error when `Name` is empty, as it should probably be required. @szv What do you think? 🤔

Comment (szv) [reply ProjectSBS/ProjectSBS/Presentation/CustomMarkupExtensions/Localize.cs:14]: Yes your are right, returning an empty string doesn't make much sense 😅

Comment (szv) [reply ProjectSBS/ProjectSBS/Presentation/CustomMarkupExtensions/Localize.cs:14]: Your could probably also introduce another optional property in the Markup Extension for formatting arguments...
https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.localization.istringlocalizer.item?view=dotnet-plat-ext-8.0#microsoft-extensions-localization-istringlocalizer-item(system-string-system-object())

Comment (morning4coffe-dev) [reply ProjectSBS/ProjectSBS/Presentation/CustomMarkupExtensions/Localize.cs:14]: Sorry for no response for a while, many things happening with graduation. Thanks for the suggestion! I am gonna merge this PR for now and create an issue for me to implement this in the future 👍

**Pull request merged**